src/containers/booklist.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable react/jsx-key */
+
+import { connect } from 'react-redux';
+import Book from '../components/book';
+import CategorySelect from '../components/categorySelect';
+
+const booklist = (props) => {
+  let books = [];
+  for (const [key, value] of Object.entries(props.books)) {
+    books.push([[key], value]);
+  }
+  books = books.map((newBook) => <Book book={newBook[1]} id={newBook[0]} />);
+
+  const noBooks = (
+    <div className="text-center no-books py-5">
+      Add books to display them here.
+    </div>
+  );
+  return (
+    <div className="book-list d-flex flex-column">
+      <div className="d-flex justify-content-between">
+        <div className="book-list-header">ALL BOOKS</div>
+        <CategorySelect />
+      </div>
+
+      {books.length === 0 ? noBooks : books}
+    </div>
```

Comment:
- [ ] Kindly let's update this to display books list in HTML table using `Book` component as per the [requirements from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/BooksList/BooksAsTable.png)
- [ ] Also let's ensure that we pass a `book` object as a prop to each `Book` component